apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -100,32 +108,192 @@ export class ChangeAction extends BaseRushAction {
         'determine which projects were changed. If this parameter is not specified, the checked out branch ' +
         'is compared against the "master" branch.'
     });
+
+    this._overwriteFlagParemter = this.defineFlagParameter({
+      parameterLongName: '--overwrite',
+      description: `If a changefile already exists, overwrite without prompting ` +
+        `(or erroring in ${BULK_LONG_NAME} mode).`
+    });
+
+    this._changeEmailParameter = this.defineStringParameter({
+      parameterLongName: '--email',
+      argumentName: 'EMAIL',
+      description: 'The email address to use in changefiles. If this parameter is not provided, the email address ' +
+        'will be detected or prompted for in intractive mode.'
+    });
+
+    this._bulkChangeParameter = this.defineFlagParameter({
+      parameterLongName: BULK_LONG_NAME,
+      description: 'If this flag is specified, apply the same change message and bump type to all changed projects. ' +
```

Comment:
What happens if different projects support different bump types?

Reply:
You'll get an error if you specify bump type that isn't supported by all projects that have changes.